### ISSUE TITLE
Reorganize Startup options in the preferences

### ISSUE THREAD

**Issue by uliska** [wbsoft/frescobaldi#231]
In the context of my "restart" additions I finally managed to try out the Sessions feature (oh my, there are quite a lot of useful things about Frescobaldi I don't know yet...). And now I realize why it didn't work as expected.

The documentation and the Preferences dialog lead me to think that if I check `Start with last used session` Frescobaldi will automatically remember the opened files and reopen them on startup.
But as I have now found out I have to create and configure a session first for this to work. Otherwise I'm in 'no session' mode which doesn't do what I expect.

https://github.com/wbsoft/frescobaldi/pull/230 documents the current behaviour.

But I suggest to change this behaviour: I would drop the 'no session' mode completely but instead have a `Default` session that is set up to "alwaqys save the list of files" and doesn't have a base directory.

When the option `Start with no session` is checked Frescobaldi should still start with the `Default` session but clear it before loading. Maybe also rename it to `Start with empty session`.

This change would allow a new user to start with the `Start with last used session` right away without having to explicitly creating and configuring a session first.

This is a 'feature request', and I won't go into it myself ;-)

**Comment by proski** [comment]
I was about to report the same issue. "Start with last used session" behaves not in a way users would expect. Named sessions are saved on exit, but unnamed sessions are not. They should be. Then it could be the default option.

Moreover, the unnamed session should be saved even if "Start with no session" is selected. The user should be able to go to Sessions menu, select "Last session" and have it restored.

Session->No session is confusing. It doesn't close the files as restarting Frescobaldi with "Start with no session" would do. It's actually creating an unnamed session from the current session.

Session->Save As is badly missing. Maybe it's misnamed as something else.

The default directory and the include path are only available to the named sessions. We should probably redesign it.

**Comment by wbsoft** [comment]
I agree, the session stuff could be organised simpler. Let me explain how I wanted/want it to be organized. For casual users the session stuff is not needed. Just like OpenOffice or MS Word don't have it. That's the difference between Frescobaldi and a developer's IDE: Frescobaldi is tailored to basic users. But for advanced users, Frescobaldi should also have all the flexibility that is needed.

I will develop a new proposal for organisation of the sessions stuff.

**Comment by PeterBjuhr** [comment]
>Session->Save As is badly missing. Maybe it's misnamed as something else.

If I'm not mistaken, you can rename a session but you can't duplicate it first as 'save as' would imply.

**Comment by wbsoft** [comment]
You could go to 'No Session' and then Save. But you'd loose specific session settings. You can also change the name in the Sessions Manager, using Edit.

**Comment by fedelibre** [comment]
> This change would allow a new user to start with the `Start with last used session` right away without having to explicitly creating and configuring a session first.

I agree, this is how browsers work and it's something usually expected by users.
I would rename the preferences tab from Sessions to Startup and offer the following options:

- Start with empty file
- Start with last opened files
- Start with Session [dropdown list]
